### DISCUSSION TITLE
iOS 报错dyld: Symbol not found: _OBJC_CLASS_$_MTLCaptureDescriptor

### DISCUSSION THREAD

**Discussion by guoaobao** [google/filament#6319] [Q&A]
![image](https://user-images.githubusercontent.com/17892094/203920556-82361365-0e04-48d2-9d3b-363851c18ab8.png)
![image](https://user-images.githubusercontent.com/17892094/203922637-a562f607-ff3b-4036-998d-74521eda8db7.png)

project : gltf-viewer OpenGL ,  simulator version ： 12.4 ,simulator version ： 15.4

**Comment by bejado** [comment]
Make sure the `iOS Deployment Target` is set to 12.4 or less in your Xcode project. It should be weak-linking the `MTLCaptureDescriptor`.

I'm unable to reproduce the second screenshot. The OpenGL backend is not recommended on iOS.
